src/app/ui/components/containers/headers/header.tsx

```diff
@@ -42,6 +42,7 @@ export function Header({
       justifyContent="center"
       margin={{ base: 0, md: 'auto' }}
       p={variant === 'bigTitle' ? 'space.05' : 'space.04'}
+      paddingLeft={variant === 'onboarding' ? 0 : undefined}
```

Comment:
I'm not proud of this variant logic. 

We went through this exercise of improving the app to have global re-usable headers but we still need to add bespoke cases all the time to match the designs

For these two column page designs, the page content is wider than the other pages to fit the mnemonic form

![Screenshot 2024-03-28 at 15 36 28](https://github.com/leather-wallet/extension/assets/2938440/9c3fb392-79e4-4ce8-806b-edf0422425ef)

I am removing left padding but it still won't match up completely as it doesn't in the design either

Reply:
> We went through this exercise of improving the app to have global re-usable headers but we still need to add bespoke cases all the time to match the designs

My understanding is that this border should be added to the page's main container as a general pattern, essentially everywhere aside from the home screen. If you're finding yourself adding bespoke logic, let's pause and discuss with the designers since I definitely don't want to slide back into a bunch of conditional styles throughout the app.

cc @fabric-8 @mica000

Reply:
In this comment I'm talking about the `Back` arrow at the top of the page which was aligned exactly as per design but doesn't align with the left content below.  

Maybe the left content should have some left padding instead? 

The onboarding pages have a different width than the others to account for the larger mnemonic key form but this means the global header doesn't line up. 

An alternative fix would be to standardise the page container width so all pages are the same width and add more padding to the home page

Reply:
This looks great @pete-watters — Agree with you, I don't quite understand right now why we aren't using the same widths and paddings across onboarding + app so we can standardize this more — I might miss something here, will sync up with @mica000. 

Would you prefer to handle the standardization bit right away in here or ok to merge this in now and handle the container standardization as a seperate issue?

Reply:
Let's not worry about more changes now. I think we have it working OK and we already spent a lot of time on this issue. 

We can think about it and maybe improve it in the future / when working on other things have it in mind